exporter/prometheusexporter/collector.go

```diff
@@ -195,7 +199,23 @@ func (c *collector) convertDoubleHistogram(metric pdata.Metric) (prometheus.Metr
 		points[bucket] = cumCount
 	}
 
-	m, err := prometheus.NewConstHistogram(desc, ip.Count(), ip.Sum(), points, attributes...)
+	arrLen := ip.Exemplars().Len()
+	var exemplarArr = make([]*dto.Exemplar, arrLen)
+	for i := 0; i < arrLen; i++ {
+		e := ip.Exemplars().At(i)
+		value := e.DoubleVal()
+		var labelPairs []*dto.LabelPair
```

Comment:
Can we use `.Len()` on `e.FilteredAttributes()` here to avoid unnecessarily appending to the slice?